### PR TITLE
Run weekly update with latest schemes

### PR DESCRIPTION
**Pull request by JamyGolden**
### Purpose of This Pull Request

To update the schemes weekly. This is actually setup for another PR which will create a basically gallery github-pages file similar to https://tinted-theming.github.io/base16-gallery/.

From the matrix channel: 
image

From my understanding this repo is the closest thing to a gallery that's why I'll submit the other PR to create the github-pages page.

- [ ] Documentation update
- [ ] Bug fix
- [ ] New feature
- [ ] Other (please explain):

### Overview of Changes

I ran against the schemes with `builder-go` on `tinted-theming/schemes/base24` and these are the changes it generated. The github action won't work for you since it's setup for tinted-theming-bot and it doesn't have access to this repo. So if you give me the info you want me to update I can do that and then you just add the personal access token to this repos secrets yourself.

Edit: I'm not able to create the other pull request since it's based on this one (github limitation with forks), but you can have a look at the diff of the commit here: https://github.com/JamyGolden/base24-html/commit/833c261d203c6ecf24546cbc8b5369835a67f42f.

It copies the output/previews directory, removes some items from the html since it takes up visual space and it's not necessary for this page. After removing that it takes a screenshot of each page and generates an html file with a page full of ``. So it's just a quick way to get something up now, doesn't have to be long lasting if we want to replace it with something else.

### CODE REVIEW

File: .github/workflows/update.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 0" # https://crontab.guru/every-week
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch the repository code
+        uses: actions/checkout@v3
```
Comment (FredHappyface) [inline .github/workflows/update.yml]: Minor note that the latest is actions/checkout@v4. Not sure if there is a massive difference or compelling reason to use the newest version

**Review by FredHappyface — APPROVED** [review]
Looks great to me, made a comment on the gh actions version but this shouldn't be blocking. As an aside I'll have a look at updating the python builder soon™

**Comment by JamyGolden** [comment]
I've updated the PR so it's using the latest github actions. I don't think this PR will work though (looking at it now) since tinted-theming-bot won't be able to make any commits if there are version changes since this repo isn't within the tinted-theming github org.

Having said that, someone actually created something like this for base24 about a week ago: https://nico-i.github.io/scheme-viewer/base24/ (https://github.com/tinted-theming/schemes/issues/22). I do like the thought of having a gallery contain all of the system variants so it's all in one place instead of in different repos.

**Comment by FredHappyface** [comment]
Happy to move this to tinted theming, I'm not precious about keeping stuff under this org. Particularly if it's blocking

**Pull request closed without merging**